client/src/main/scala/com/gu/anghammarad/Anghammarad.scala

```diff
@@ -10,28 +10,60 @@ import scala.concurrent.{ExecutionContext, Future}
-    * @param channel      The notification channel you'd like to use
-    * @param target       Specify who should receive the message
     * @param actions      Specify Call To Action buttons that will be put at the end of an email / hangout message
+    * @param target       Specify who should receive the message
+    * @param channel      The notification channel you'd like to use
+    * @param sourceSystem The system sending the notification (your system)
     * @param topicArn     ARN of Anghammarad's SNS topic (you will need to obtain this and put it in your app's config)
     * @param client       The SNS client that should be used to add your notification to the topic
-    * @return             Future of the resulting SNS Message ID
+    * @return             Future containing the resulting SNS Message ID
     */
-  def notify(subject: String, message: String, sourceSystem: String,
-             channel: RequestedChannel, target: List[Target], actions: List[Action],
-             topicArn: String,
-             client: AmazonSNSAsync = AWS.snsClient(AWS.credentialsProvider()))
+  def notify(subject: String, message: String, actions: List[Action], target: List[Target], channel: RequestedChannel,
+             sourceSystem: String, topicArn: String, client: AmazonSNSAsync = defaultClient)
             (implicit executionContext: ExecutionContext): Future[String] = {
     val request = new PublishRequest()
       .withTopicArn(topicArn)
       .withSubject(subject)
       .withMessage(messageJson(message, sourceSystem, channel, target, actions))
     awsToScala(client.publishAsync)(request).map(_.getMessageId)
   }
+
+  /**
+    * Sends a notification using Anghammarad.
+    *
+    * @param notification The notification to send
+    * @param topicArn     ARN of Anghammarad's SNS topic (you will need to obtain this and put it in your app's config)
+    * @return             Future containing the resulting SNS Message ID
+    */
+  def notify(notification: Notification, topicArn: String)
+            (implicit executionContext: ExecutionContext): Future[String] = {
+    val request = new PublishRequest()
+      .withTopicArn(topicArn)
+      .withSubject(notification.subject)
+      .withMessage(messageJson(notification.message, notification.sourceSystem, notification.channel, notification.target, notification.actions))
+    awsToScala(defaultClient.publishAsync)(request).map(_.getMessageId)
+  }
+
+  /**
+    * Sends a notification using Anghammarad.
+    *
+    * @param notification The notification to send
+    * @param topicArn     ARN of Anghammarad's SNS topic (you will need to obtain this and put it in your app's config)
+    * @param client       The SNS client that should be used to add your notification to the topic
+    * @return             Future containing the resulting SNS Message ID
+    */
+  def notify(notification: Notification, topicArn: String, client: AmazonSNSAsync)
```

Comment:
Scala doesn't allow multiple overloaded methods to have default parameters, so there are two versions of this notify. One with a client, and one that uses the default.

Another approach would be to come up with different names for each way of calling `notify`, but naming is hard.